dotNETStandard/HealthVault.Mobile/cs/Xamarin.Core/ViewModels/MedicationsMainViewModel.cs

```diff
@@ -98,7 +112,7 @@ private void UpdateDisplay(IEnumerable<Medication> items)
         {
             var medicationsMainPage = new MedicationsSummaryPage()
             {
-                BindingContext = new MedicationsSummaryViewModel(medication, this.ingredientChoices, this.thingClient, this.recordId, this.NavigationService),
+                BindingContext = new MedicationsSummaryViewModel(medication, this.connection, this.NavigationService),
```

Comment:
I dont think that this project requires IoC container. It will only complicate things.

---
In reply to: [117601250](https://github.com/Microsoft/healthvault-samples/pull/11#discussion_r117601250) [](ancestors = 117601250,117592508)